nucypher/blockchain/eth/sol/source/contracts/lib/Bits.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.6.1;
+
```

Comment:
Are we using this as a linked library - or how does this get included at compile time?

Reply:
we compile all files from lib and connect this to `StakingEscrow` in import statement